### PR TITLE
Config simulation

### LINKED ISSUE #2: Fix the mess in the launch file between the different branches

### PR DESCRIPTION
**Pull request by Kam0ss**
## Description
Added a different motors configuration for the simulation. Otherwise the sub in the simulation would just go up.
Had to change the way the configuration file was read and change the script to launch the node and also create a separate script to launch the simulation.
Also added a script to take msg from effortThruster to turn them into csv.
Fixed an issue to prevent the motors to go higher than 30% and lower than -30%.
**Need to make a change in the launch script.**

## Issue link
#2 

## How has this been tested ?
Tested the simulation, gonna test it in the pool.

## Checklist
- [?] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: scripts/ThrusterEffortMsgToCsv.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+        rate = rospy.Rate(100)
+
+        while not rospy.is_shutdown():
+            rate.sleep()
+
+        self.csv_file.close()
+
+    def write_header(self):
+        # self.write_line("seq;stamp;frequency;amplitude;noise;heading;elevation;raw_data_chanRefReal;raw_data_chanRefImage;raw_data_chan1Real;raw_data_chan1Image;raw_data_chan2Real;raw_data_chan2Image")
```
Comment (gauthiermartin) [inline scripts/ThrusterEffortMsgToCsv.py]: This should be removed if it's not used

File: scripts/ThrusterEffortMsgToCsv.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+    def callback(self, object):
+
+        ID = object.ID
+        effort = object.effort
+
+        elements = [ID, effort]
+
+        #seq = object.header.seq
```
Comment (gauthiermartin) [inline scripts/ThrusterEffortMsgToCsv.py]: Same as previous

File: src/provider_thruster/provider_thruster_node.cc [hunk 3]
```diff
@@ -22,15 +24,18 @@ namespace provider_thruster {
         calcul(8,6)
     {
 
-        YAML::Node node = YAML::LoadFile(file_path_);
+        //YAML::Node node = YAML::LoadFile(file_path_);
```
Comment (gauthiermartin) [inline src/provider_thruster/provider_thruster_node.cc]: Same as previous

File: src/provider_thruster/provider_thruster_node.cc [hunk 4]
```diff
@@ -22,15 +24,18 @@ namespace provider_thruster {
...
 
         int j=0;
         for(auto&t : fichier){
-            auto thruster = node[t];
-            assert(thruster.Type() == YAML::NodeType::Sequence);
+            std::vector<double> value;
+            ros::param::get("/provider_thruster/" + t, value);
+
+            //auto thruster = node[t];
+            //assert(thruster.Type() == YAML::NodeType::Sequence);
```
Comment (gauthiermartin) [inline src/provider_thruster/provider_thruster_node.cc]: Removed dead code

File: src/provider_thruster/provider_thruster_node.h [hunk 5]
```diff
@@ -56,7 +56,7 @@ class ProviderThrusterNode {
   ros::Publisher rs485Publisher;
   ros::Publisher effortPublisher;
 
-  const std::string file_path_ = std::string(std::getenv("ROS_SONIA_WS") ) + "/src" + "/provider_thruster" + "/config" + "/thruster.yaml";
+  //const std::string file_path_ = std::string(std::getenv("ROS_SONIA_WS") ) + "/src" + "/provider_thruster" + "/config" + "/thruster.yaml";
```
Comment (gauthiermartin) [inline src/provider_thruster/provider_thruster_node.h]: Same as previous

**Review by gauthiermartin — CHANGES_REQUESTED** [review]
Please remove dead code

**Review by Kam0ss — COMMENTED** [review]
Dead code removed.

**Review by gauthiermartin — APPROVED** [review]
Seem good to me

**Pull request merged**